src/cloudfront/origin-access.spec.ts

```diff
@@ -31,8 +31,7 @@ describe('upsertOriginAccessControl', () => {
     const distributionId = 'my-distribution-id'
     const oacName = getOriginAccessControlName(domainName, distributionId)
 
-    listOriginAccessControlsMock.mockReturnValue(
-      awsResolve({
```

Comment:
Is `awsResolve` still required now that we have "real" promise ?

Reply:
Well, we could use `Promise.resolve(x)` instead directly, but maybe we want to keep this abstraction in case they decide to change the way it works again ? It was handy to have it here and change it only in one place this time

Reply:
Okay, thank you for the explanation